### PR TITLE
Increase maximum featured documents for No. 10 to 7

### PR DESCRIPTION
**Pull request by KludgeKML**
https://trello.com/c/ey4KM89d/59-make-changes-to-no10-organisation-page

## Before
Screenshot 2024-10-16 at 14 15 53

## After
Screenshot 2024-10-16 at 14 15 41

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/models/organisation.rb [hunk 1]
```diff
@@ -112,6 +112,12 @@ class Organisation < ApplicationRecord
 
   has_many :offsite_links, as: :parent
 
+  def featured_documents_display_limit
```
Comment (leenagupte) [inline app/models/organisation.rb:115]: I'd have thought this method would need to be updated: [featured_documents_display_limit](https://github.com/alphagov/whitehall/blob/06a1c9e2bbbc31a0b34807f8b8b8a167b3272c53/app/models/organisation.rb#L510) ?

**Review by leenagupte — COMMENTED** [review]
I have an inline question.

File: app/models/organisation.rb (hunk 1 above)
Comment (KludgeKML) [reply app/models/organisation.rb:115]: Lol, that's what Ashley thought as well. Featured links and featured documents are two separate things.

Comment (leenagupte) [reply app/models/organisation.rb:115]: Oh! That's my fault, Ash got that from my investigations. Oops!

**Pull request merged**